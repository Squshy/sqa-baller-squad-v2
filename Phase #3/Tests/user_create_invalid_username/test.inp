CREATE
COW BOYS ON THEY HORSES