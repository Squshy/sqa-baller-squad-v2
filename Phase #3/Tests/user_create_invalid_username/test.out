=========================================================
                        CREATE
=========================================================

Enter a username: COW BOYS ON THEY HORSES

---------------------------------------------------------
          Username must be 15 characters or less.
=========================================================






